app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
         N_("Create a filter rule. The parameters included should be based upon the filter type.")
     param :content_view_filter_id, :identifier, :desc => N_("filter identifier"), :required => true
     param :name, String, :desc => N_("package or package group: name")
+    param :names, Array, :desc => N_("packages or package groups: names")
```

Comment:
You'd have to ask someone from the Apipie project if it supports dual types for documentation. From a code perspective, we could easily support both types while deprecating one.